testing/scenario_app/android/README.md

```diff
@@ -16,23 +16,46 @@ $ ./testing/scenario_app/run_android_tests.sh android_debug_unopt
 $ ./testing/scenario_app/run_android_tests.sh android_debug_unopt_arm64
 ```
 
+## Debugging
+
+Debugging the tests on CI is not straightforward but is being improved:
+
+- <https://github.com/flutter/flutter/issues/143458>
+- <https://github.com/flutter/flutter/issues/143459>
+
+Locally (or on a temporary PR for CI), you can run the tests with the
+`--smoke-test` argument to run a single test by class name, which can be useful
+to verify the setup:
+
+```sh
+# From the root of the engine repository
+$ ./testing/scenario_app/run_android_tests.sh android_debug_unopt_arm64 --smoke-test dev.flutter.scenarios.EngineLaunchE2ETest
+```
+
+The result of `adb logcat` and screenshots taken during the test will be stored
+in a logs directory, which is either `FLUTTER_LOGS_DIR` (on CI) or locally in
```

Comment:
nonblocking nit: instead of (on ci) I think it should be (if set like in ci)

Reply:
Done